File: Assignment_16/Assignment4.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0, j = 0;

    for(i = 1; i <= iRow; i++)
    {
        for(j = 1; j <= iCol; j++)
        {
            if((i == 1) ||( i == iCol)||(j == 1)||(j == iCol)||(i==j))
            {
                printf("*\t");
            }
            else
            {
                printf("\t");
            }
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;


    printf("Enter The Number Of Rows :");
    scanf("%d",&iValue1);

    printf("Enter The Number Of Colums :");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);


    return 0;
}

File: Assignment____46/Assignment4.java
//  Output :
//  4   4   4   4
//  3   3   3   3
//  2   2   2   2
//  1   1   1   1


import java.util.*;

class Pattern
{
    public void Pattern(int iRow , int iCol)
    {
        for(int i = iRow; i >= 1; i--) 
        {
            for(int j = 1; j <= iCol; j++)
            {
                System.out.print(i + "\t");
            }
            System.out.println();
        }
    }
}
class Assignment4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number of Rows : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter Number of Columns : ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1 , iValue2);

    }
}